dev/column_result.h

```diff
@@ -10,12 +10,23 @@
 #include "rowid.h"
 #include "alias.h"
 #include "column.h"
+#include "column_traits.h"
 #include "storage_traits.h"
 
 namespace sqlite_orm {
     
     namespace internal {
-        
+
+        template<class O, class C>
+        typename column_traits<C>::field_type invoke_column(const O& o, const C& c, std::false_type) {
```

Comment:
where these two functions are used? If nowhere please remove them

Reply:
@fnc12  resolved in #329